NetCord/EmbedAuthor.cs

```diff
@@ -18,7 +18,7 @@ public EmbedAuthor(JsonModels.JsonEmbedAuthor jsonModel)
     public string? Name => _jsonModel.Name;
     
     /// <summary>
-    /// When set, turns the <see cref="Name"/> into a clickable link, pointing to the specified URL.
+    /// When set, turns the <see cref="Name"/> into a clickable link, pointing to the base of the specified URL.
```

Comment:
The URL doesn't point to the base of the specified URL. For example for youtube it points to the video author. I think the previous wording was better.

Reply:
Will revert then

Reply:
Done and done